scene/resources/texture.cpp

```diff
@@ -1843,6 +1884,9 @@ void AnimatedTexture::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_frame_delay", "frame"), &AnimatedTexture::get_frame_delay);
 
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "frames", PROPERTY_HINT_RANGE, "1," + itos(MAX_FRAMES), PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_UPDATE_ALL_IF_MODIFIED), "set_frames", "get_frames");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "current_frame"), "set_current_frame", "get_current_frame");
```

Comment:
I'm worried that saving the current frame would mean that the scene is modified every time (due to it being saved on a different frame). Maybe giving it property usage `0`, so that it is accessible to script, but is not saved would do?

Alternatively, it might be possible to give it property usage of `0` in `_validate_property`, so that the frame is editable only when `pause` is `true`.

Reply:
fixed